helm/dex-app/templates/podlogs.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if and .Values.monitoring .Values.monitoring.podLogs .Values.monitoring.podLogs.enabled }}
+---
+# PodLogs for dex
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  name: dex-logs
+  namespace: giantswarm
+  labels:
+    app.kubernetes.io/name: dex
+  annotations:
+    giantswarm.io/notes: "Log collection for DEX identity provider"
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: dex
+      app.kubernetes.io/component: dex
+      app.kubernetes.io/instance: dex-app
+  relabelings:
+  - action: replace
+    replacement: giantswarm
+    targetLabel: giantswarm_observability_tenant
+  - action: replace
+    sourceLabels: [__meta_kubernetes_pod_node_name]
+    targetLabel: node_name
+  - action: replace
+    replacement: dex
+    targetLabel: component
+
+---
+# PodLogs for dex-operator
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  name: dex-operator-logs
+  namespace: giantswarm
+  labels:
+    app.kubernetes.io/name: dex-operator
+  annotations:
+    giantswarm.io/notes: "Log collection for DEX operator"
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: dex-operator
+      app.kubernetes.io/instance: dex-operator
+  relabelings:
+  - action: replace
+    replacement: giantswarm
+    targetLabel: giantswarm_observability_tenant
+  - action: replace
+    sourceLabels: [__meta_kubernetes_pod_node_name]
+    targetLabel: node_name
+  - action: replace
+    replacement: dex-operator
+    targetLabel: component
```

Comment:
This looks correct but should belong to the [dex-operator app](https://github.com/giantswarm/dex-operator/)